cmd/gsw/axtHelper.go

```diff
@@ -6,18 +6,37 @@ import (
 	"github.com/vertgenlab/gonomics/fileio"
 	"github.com/vertgenlab/gonomics/simpleGraph"
 	"github.com/vertgenlab/gonomics/vcf"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
 )
 
 func convertAxt(axtFile, format, targetFa, output string) {
 	switch format {
 	case "vcf":
```

Comment:
It looks like this now converts and sorts.  Maybe convertAxt should just convert and then sort it afterwards?